### PR TITLE
Updated the cat url.

### PR DESCRIPTION
**Pull request by auguwu**
**Please describe the changes this PR makes and why it should be merged, as well as any and all proof of successful tests:**
It just changes the cat api url and no space after the variables and the class.

**Semantic versioning classification:**
I don't even know :^)

- [ ] This PR changes the framework's interface (methods or parameters added)
  - [ ] This PR includes breaking changes (methods removed or renamed, parameters moved or removed)
- [x] This PR **only** includes non-code changes, like changes to README, etc.

### CODE REVIEW

File: commands/animals/cat.js [hunk 1]
```diff
@@ -23,8 +24,7 @@ class Cat extends Social {
       }
 
       const msg = await message.channel.send(`<a:typing:397490442469376001> **${message.member.displayName}** is petting a cat...`);
-      const { body } = await snekfetch.get("https://api.weeb.sh/images/random?type=animal_cat")
-        .set("Authorization", `Wolke ${process.env.WEEBSH}`)
+      const { body } = await snekfetch.get("https://nekos.life/api/v2/img/meow")
         .set("User-Agent", `Misaki/${version}/${this.client.user.id === "396323622953680910" ? "Production" : "Development"}`);
```
Comment (YorkAARGH) [inline commands/animals/cat.js]: You forgot to remove this line since you're not using `weeb.sh` anymore

Comment (auguwu) [reply commands/animals/cat.js]: Ok

**Comment by MrJacz** [comment]
Please do not use the online editor to do PR's and make sure to run eslint before committing.
It seems the weeb.sh api is still perfectly fine so what is the purpose of this PR?

**Comment by auguwu** [comment]
1) Ok
2) ask York, he wanted to use another API.

**Comment by YorkAARGH** [comment]
I never said I wanted to use another `cat` api, I said I wanted more `nsfw` api's

**Comment by auguwu** [comment]
oh

**Pull request closed without merging**